packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -274,7 +274,20 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
       };
     }
 
-    const Content = (
+    const Button = () => (
```

Comment:
If you made `keytipProps` an arg to this function then could you move `Tag` in here? (In one case, it would be undefined, but that should be ok for the spread.)